front/app/components/FilterBoxes/StatusFilter.tsx

```diff
@@ -30,36 +38,16 @@ const Count = styled.span`
   `}
 `;
 
-const CloseIcon = styled(Icon)`
-  fill: #fff;
-  margin-left: auto;
-
-  ${isRtl`
-    margin-left: 0;
-    margin-right: auto;
-  `}
-`;
-
 const Status = styled.button`
-  color: ${({ theme }) => theme.colors.tenantText};
-  font-size: ${fontSizes.base}px;
-  font-weight: 400;
-  line-height: normal;
   display: flex;
+  cursor: pointer;
   align-items: center;
   justify-content: space-between;
-  padding-left: 18px;
-  padding-right: 18px;
-  padding-top: 7px;
-  padding-bottom: 7px;
-  margin: 0px;
-  margin-right: 10px;
-  margin-bottom: 6px;
-  cursor: pointer;
-  border-radius: 5px;
-  user-select: none;
-  transition: all 80ms ease-out;
```

Comment:
Removed some of the styling, but I'll add a larger refactor of this component to the "Nice to have" ticket :)